app/questionnaire/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+
+from sports.models import Sport
+
+
+class Question(models.Model):
+    text = models.CharField(max_length=100)
+
```

Comment:
Might be nice to have a  __str__ function for these models, but isn't strictly necessary